src/espina/shared/components/route/route_link.component.js

```diff
@@ -2,14 +2,18 @@
 
 import React from 'react';
 
-import template from './route_link.rt.html';
-import SpikeComponent from 'shared/base_component';
+import SpikeComponent from 'espina/shared/base_component';
 
 class RouteLinkComponent extends SpikeComponent {
 
 
   get template(){
-    return template;
+    return () => (
+        <a className={["bh-route-link", this.class]}
```

Comment:
We should try and remove all references to the basil health project (bh, basilhealth, basil-health) in both repositories.